src/Client.php

```diff
@@ -116,14 +121,39 @@ public function __construct(Builder $httpClientBuilder = null)
 
         $builder->addPlugin(new ExceptionThrower());
         $builder->addPlugin(new HistoryPlugin($this->responseHistory));
-        $builder->addPlugin(new HeaderDefaultsPlugin([
-            'User-Agent' => self::USER_AGENT,
-        ]));
+        $builder->addPlugin(
```

Comment:
Please revert this change.

Reply:
Reverted in d18e1d10fcd4f2768afcef81c011bcc29ce0c1e1